src/models/users.js

```diff
@@ -30,7 +34,7 @@ export default class extends AbstractModel {
         email: user.email,
         creation_date: user.creation_date,
         userId: user.id,
-        avatar: user.avatar || "default",
+        avatar: user.avatar || "default"
```

Comment:
Missing trailing comma . See:
https://eslint.org/docs/rules/comma-dangle

Reply:
Yup, I think running `yarn lint --fix` will fix all of these things.

Reply:
i don't know why but webstorm remove trailing comma when "reformat with prettier" ??? i will take care of this

Reply:
i have other errors on master with yarn lint : 
on src/index.js
8:20  error  Unable to resolve path to module './config'  import/no-unresolved
8:20  error  Missing file extension for "./config"        import/extensions

Reply:
and comma missing fixed